### PR TITLE
BUG: filter name issue

### PR DESCRIPTION
**Pull request by licode**
Duplicated names are shown in the filter.

### CODE REVIEW

File: databroker/tests/test_broker.py [hunk 1]
```diff
@@ -389,6 +389,11 @@ def test_filters(db_empty, RE):
 
     assert header['start']['uid'] == ken_calib_uid
 
+    db.clear_filters()
+    db.add_filter(start_time='2017')
+    db.add_filter(start_time='2017')
```
Comment (tacaswell) [inline databroker/tests/test_broker.py]: what does this do if you now add `db.add_filter(start_time='2018')`?

Comment (licode) [reply databroker/tests/test_broker.py]: OK, I think it is better to replace 2017 with 2018.

Comment (danielballan) [reply databroker/tests/test_broker.py]: I think you are misunderstanding Tom's question. This test is good as is for testing that duplicate are ignored. He is suggesting that you check that this doesn't make _all_ things ignored, only duplicates.

Comment (licode) [reply databroker/tests/test_broker.py]: then we will have two start_time as 2017 and 2018. I can add more tests to double check.

Comment (licode) [reply databroker/tests/test_broker.py]: I think we still need to consider the case 
```
db.add_filter(start_time='2017')
db.add_filter(start_time='2017', stop_time='2018). 
```
So basically we will raise error if there are duplicated keys. Users can clean the filters and search again. Also per discussion with @tacaswell

File: databroker/broker.py [hunk 2]
```diff
@@ -433,7 +433,12 @@ def add_filter(self, **kwargs):
         :meth:`Broker.clear_filters`
 
         """
-        self.filters.append(dict(**kwargs))
+        d = dict(**kwargs)
+        for val in self.filters:
+            for k,v in val.items():
```
Comment (danielballan) [inline databroker/broker.py]: Since you don't need the keys, `val.values()` would be better here.

File: databroker/broker.py [hunk 3]
```diff
@@ -433,7 +433,12 @@ def add_filter(self, **kwargs):
...
+                if k in d.keys():
```
Comment (danielballan) [inline databroker/broker.py]: `k in d` is equivalent to `k in d.keys()` and faster in Python 2 because it avoids creating an intermediate list.

Comment (licode) [reply databroker/broker.py]: thanks, good to know.

File: databroker/broker.py [hunk 4]
```diff
@@ -433,7 +433,12 @@ def add_filter(self, **kwargs):
...
+                    raise ValueError('Filter key already exists.')
```
Comment (tacaswell) [inline databroker/broker.py]: Can you include in the error message what the filter it conflicts with is and how to fix the problem?

**Comment by tacaswell** [comment]
Replaced by #269

**Pull request closed without merging**